batch/sql/estimated-current.sql

```diff
@@ -263,6 +263,7 @@ CREATE TABLE IF NOT EXISTS `jobs` (
   `inst_coll` VARCHAR(255),
   `n_regions` INT DEFAULT NULL,
   `regions_bits_rep` BIGINT DEFAULT NULL,
+  `time_ready` BIGINT DEFAULT NULL,
```

Comment:
I feel uneasy about adding this kind of metadata to the `jobs` table, it feels like a slippery slope for adding more and more telemetry and muddling the purpose of this table. Why not add a separate table for this?

Something that I think is important with telemetry is that as we track more and more things we might need to consider not turning tracking on for every request/resource in the system. I think we should design for eventually sampling or toggling telemetry instead of locking ourselves into tracking certain metadata forever. In this setup, I can't really tell the difference between a job that is not ready yet and a job for which we are not tracking this telemetry. With a separate table, we can just not add a row for a job we're not tracking.

Reply:
Another note: I think it's worth thinking about this kind of telemetry in terms of state change and *event* tracking. I think it's worth considering whether instead of having a `time_ready` column we should have some table that looks like having `(prev_state, new_state, timestamp)`.

Reply:
Good suggestion, but let's do that another time with a more fully fleshed out overall plan of monitoring the system?